src/Goerli-DssSpellCollateral.sol

```diff
@@ -32,57 +70,176 @@ contract DssSpellCollateralAction {
+
+    // https://gateway.pinata.cloud/ipfs/QmRLwB7Ty3ywSzq17GdDdwHvsZGwBg79oUTpSTJGtodToY
+    string  internal constant RWA007_DOC                     = "QmRLwB7Ty3ywSzq17GdDdwHvsZGwBg79oUTpSTJGtodToY";
+    // There is no DssExecLib helper, so WAD precision is used.
+    uint256 internal constant RWA007_A_INITIAL_PRICE         = 250_000_000 * WAD;
+    uint48  internal constant RWA007_A_TAU                   = 0;
+
+    // Ilk registry params
+    uint256 internal constant RWA007_REG_CLASS_RWA           = 3;
+
+    // Remaining params
+    uint256 internal constant RWA007_A_LINE                  = 1_000_000;
+    uint256 internal constant RWA007_A_MAT                   = 100_00; // 100% in basis-points
+    uint256 internal constant RWA007_A_RATE                  = ZERO_PCT_RATE;
+
+    // Monetalis operator address
+    address internal constant RWA007_A_OPERATOR              = 0x94cfBF071f8be325A5821bFeAe00eEbE9CE7c279;
+    // Coinbase custody address
+    address internal constant RWA007_A_COINBASE_CUSTODY      = 0xC3acf3B96E46Aa35dBD2aA3BD12D23c11295E774;
+
+    // -- RWA007 END --
+
+    function onboardRwa007(
+        ChainlogAbstract CHANGELOG,
+        IlkRegistryAbstract REGISTRY,
+        address MIP21_LIQUIDATION_ORACLE,
+        address MCD_VAT,
+        address MCD_JUG,
+        address MCD_SPOT,
+        address MCD_JOIN_DAI,
+        address MCD_PSM_USDC_A
+    ) internal {
+        // RWA007-A collateral deploy
+        bytes32 ilk      = "RWA007-A";
+        uint256 decimals = GemAbstract(RWA007).decimals();
+
+        // Sanity checks
+        require(GemJoinAbstract(MCD_JOIN_RWA007_A).vat()                             == MCD_VAT,                 "join-vat-not-match");
+        require(GemJoinAbstract(MCD_JOIN_RWA007_A).ilk()                             == ilk,                     "join-ilk-not-match");
+        require(GemJoinAbstract(MCD_JOIN_RWA007_A).gem()                             == RWA007,                  "join-gem-not-match");
+        require(GemJoinAbstract(MCD_JOIN_RWA007_A).dec()                             == decimals,                "join-dec-not-match");
+
+        require(RwaUrnLike(RWA007_A_URN).vat()                                       == MCD_VAT,                 "urn-vat-not-match");
+        require(RwaUrnLike(RWA007_A_URN).jug()                                       == MCD_JUG,                 "urn-jug-not-match");
+        require(RwaUrnLike(RWA007_A_URN).daiJoin()                                   == MCD_JOIN_DAI,            "urn-daijoin-not-match");
+        require(RwaUrnLike(RWA007_A_URN).gemJoin()                                   == MCD_JOIN_RWA007_A,       "urn-gemjoin-not-match");
+        require(RwaUrnLike(RWA007_A_URN).outputConduit()                             == RWA007_A_OUTPUT_CONDUIT, "urn-outputconduit-not-match");
+        require(RwaUrnLike(RWA007_A_URN).wards(address(this))                        == 1,                       "pause-proxy-not-relyed-on-urn");
```

Comment:
hmm, not sure about the `wards` check here, it doesn't follow previous sanity checks patterns, that should cover constructor args mainly.

Reply:
I add this check simply because this ward was added directly by the deploy script